.circleci/config.yml

```diff
@@ -101,7 +116,8 @@ workflows:
           filters:
             tags:
               only: /.*/
-      - build_extension
+      - build_extension:
+          context: Honeycomb Secrets for Public Repos
```

Comment:
I _think_ you need to specify a tag filter for `build_extension` because it's a dependency of the publish jobs, which have tag filters: https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag

If we care about forked PRs not failing due to context permissions, you can use this to exclude forked builds: https://github.com/honeycombio/opentelemetry-exporter-python/blob/main/.circleci/config.yml#L191
(notice we need both tag filter and branch filter)

Or, maybe we only want to build extension on versioned tags.

Reply:
@vreynolds thank you! Done. :+1: